tbxforms/layout/fields.py

```diff
@@ -421,16 +425,26 @@ def add_attributes(self, **kwargs):
 
         Args:
             **kwargs: keyword arguments that will be added as HTML attributes.
+        """
 
-        Returns:
+        # Setup conditional attributes.
+        kwargs = setup_conditional_attrs(attrs=kwargs)
 
-        """
-        self.attrs.update(
-            {
-                k.replace("_", "-"): conditional_escape(v)
-                for k, v in kwargs.items()
-            }
-        )
+        for k, v in kwargs.items():
+            key = k.replace("_", "-")
+
+            # Don't escape values that are already JSON encoded
+            if k in [
+                "data-conditional-field-name",
+                "data-conditional-field-values",
+            ]:
+                value = v
+            else:
+                value = conditional_escape(v)
+
+            self.attrs.update({key: value})
```

Comment:
I _think_ now you can also replace this whole for-loop with just this: `self.attrs.update(kwargs)`

Because `flatatt` will do the escaping and the dashes too.

Reply:
BOOM, yep!